### PR TITLE
[FrameworkBundle] Deprecate `framework:exceptions` XML tag

### PR DESCRIPTION
**Pull request by MatTheCat**
| Q             | A
| ------------- | ---
| Branch?       | 6.3
| Bug fix?      | no
| New feature?  | no
| Deprecations? | yes
| Tickets       | N/A
| License       | MIT
| Doc PR        | N/A

Follow-up of #48259

### CODE REVIEW

File: UPGRADE-6.3.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ DependencyInjection
 
  * Deprecate `PhpDumper` options `inline_factories_parameter` and `inline_class_loader_parameter`, use `inline_factories` and `inline_class_loader` instead
 
+FrameworkBundle
+---------------
+
+ * Deprecate `framework:exceptions` tag, unwrap it and replace `framework:exception` tags’ `name` attribute by `class`.
```
Comment (fabpot) [inline UPGRADE-6.3.md]: Can we add a before/after example?

**Review by fabpot — CHANGES_REQUESTED** [review]

File: UPGRADE-6.3.md (hunk 1 above)
Comment (MatTheCat) [reply UPGRADE-6.3.md]: Yes! https://github.com/symfony/symfony/pull/48665/commits/39be25d2e76453ca9b6f554e701cbfe4b7126fe3

Comment (nicolas-grekas) [reply UPGRADE-6.3.md]: final dot should be removed + let's use ASCII: `s/’/'/`

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -1202,6 +1202,8 @@ private function addExceptionsSection(ArrayNodeDefinition $rootNode)
                                 return $v;
                             }
 
+                            trigger_deprecation('symfony/framework-bundle', '6.3', '"framework:exceptions" tag is deprecated. Unwrap it and replace your "framework:exception" tags’ "name" attribute by "class".');
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php]: `s/’/'/`

**Review by nicolas-grekas — APPROVED** [review]

**Review by nicolas-grekas — COMMENTED** [review]
Pleas add a changelog entry also

**Comment by MatTheCat** [comment]
All done :ok_hand:

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @MatTheCat.

**Pull request merged**